pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
```

Comment:
I mentioned it before, but yes I think we should leave it up to the backend about what goes into the sdist, because the backend is the place that is handling this already in this PEP (unless we want to drastically alter this PEP). Building the wheel is *more* involved then building the sdist, not less it needs to compute almost all the same stuff, but also a bunch of other stuff. 

In addition, we're already assuming (or we should be if we're not) that the build tool is going to be having logic to produce sdists, otherwise with ``setup.py`` gone, what is producing a sdist? If we're not baking support for sdists into the standard, then build tool authors are going to be incentivized to not make it possible to produce sdists, and I am strongly -1 on any proposal that makes or incentivizes it to be meaningfully harder to produce a sdist + wheels then just wheels.